### ISSUE TITLE
Purpose of repo

### ISSUE THREAD

**Issue by brokeharvard** [tomtrask/predict-it-life-expectancy#1]
Was this code just meant to determine which PredictIt contracts were scheduled to end within a given time frame? What was the purpose of that?

I'm interested in using quantitative analysis to inform trading on PredictIt, and I'm looking for similarly minded people/projects.

**Comment by tomtrask** [comment]
Yes, sort of. I was looking for the equivalent of the "Who will leave next"
contract. Instead of one contract, you now have a set of unconnected
contracts. I also wanted to see the arbitrage possibility with two
Tillerson contracts with different expiry dates.

Somewhere on my to-do list I wanted to come up with a good graphical
representation. It seems like when there's conflict (e.g. Tillerson/Trump)
the predicted lifespans shorten for all contracts (at least cabinet
contracts are correlated, obviously Kim Jung Un isn't). I also want to
estimate confidence intervals on single contracts (e.g. SHEA.NH01.2018,
picking one totally at random). There's so many questions to answer from
this one set of data.
